app/enterprise/1.3-x/kong-cloud.md

```diff
@@ -1,12 +1,74 @@
 ---
 title: Kong Cloud
-toc: false
+toc: true
 ---
 
-## Kong Cloud Proxy TLS
+# Kong Cloud Overview
+Kong Cloud is our managed Kong Enterprise offering, based on the proven Kong Enterprise Gateway (Kong API Gateway). Fully automated, forget about scaling, data persistence, high availability configurations, and failure recovery. [Start a free trial](https://konghq.com/products/kong-enterprise/free-trial) today and end your operational worries.
 
-Kong Cloud enforces HTTPS for all of the services that it operates, including Admin API, Kong Manager, and Kong Developer Portal. Kong Cloud does not enforce HTTPS for traffic destined for the customer's proxy, since certificate management and associated domains are under the customer's control. To enforce HTTPS on upstream traffic, use the [certificate](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#certificate-object) and [SNI](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#sni-object.) objects through the Admin API.
+# Kong Cloud  differences from on-premise
+Kong Cloud, while being our hosted Kong Enterprise solution, has a few key differences from a self-hosted, on-premise Kong Enterprise cluster.
```

Comment:
Would avoid "self-hosted" and "our" in the same sentence, and avoid "on-premise" as a distinction from Cloud in the same sentence. 

```suggestion
Despite simply being a managed deployment of Kong Enterprise, Kong Cloud has a few key differences from a customer-hosted Kong Enterprise deployment.
```